### ISSUE TITLE
Can I use this script to get followers in tumblr ?

### ISSUE THREAD

**Issue by ashishkatoch** [GaneschaTeam/ngGanescha.Tumblr#1]
I want to make functionality on my website to store follower information via authentication of tumblr logged in users 

Regards

**Comment by oliver-j** [comment]
First of all, at the moment the module only allows the user to "log in" into tumblr. 
Which means you can get the tokens of the logged in user. You still have to do the server stuff on you own.

This is a small part of a ecosystem, so there is no warranty that this repo is not under heavy changes.
 And all the http posts are sent to our server address, like "/app" and uses a kind of DDL.

I am pretty sure your server doesn't use the same DLL like ours. 

TL:DR: 
I don't think so 

Regards

**Comment by ashishkatoch** [comment]
Okay Thanks for reply...Do you know any script that help me to get the followers via tumblr. Right now I have successfully fetched followers of fb,twitter,linkedin,google and pinterest via javascript API's. But tumblr and instagram are giving issue.

Regards
Ashish

**Comment by oliver-j** [comment]
If you are using NodeJS on the server side I can recommend https://github.com/tumblr/tumblr.js which has everything that the tumblr api offers.

**Comment by ashishkatoch** [comment]
I am not using nodejs that's is the main issue. All other social website provide simple js api's, but tumblr is exceptional.
